README.md

```diff
@@ -137,6 +137,22 @@ channel.subscribe(events, new MessageListener() {
 });
 ```
 
+### Subscribing to a channel with deltas ###
+
+Request a Vcdiff formatted delta stream using channel options when you get the channel:
```

Comment:
I thought we were calling this delta compression?
This description assumes way too much.  

* Does a user know what deltas are?
* Why would they use them?
* What benefit do they get?

@MarkWoulfeAbly can you get involved here perhaps as this extends to how we more generally deal with documentation?

Reply:
Having spoken with @paddybyers I believe the breakdown is as follows:

Delta Compression - The name of the feature as a whole.
Delta Mode - An attribute to your attachment of a channel.
delta - The specific name of the flag that you change.

So the title could be updated to mention delta compression instead, though I don't know if that gives too much more context. It might make sense to link through to the actual documentation here if more information is needed.

Can I ask where this ReadMe will be visible from? Just GitHub? Do we know who will be getting their information from here as opposed to the documentation?

Reply:
> Just GitHub?

Yes. But it will be what many - perhaps most - developers use to get started.

Reply:
I would be interested in understanding a bit more about GitHub as an entry point for exposing users to new features. Alongside the documentation and support help pages that make 3 potential sources of truth which can be an issue. In a case like this, it sounds like we just want to give the developer a taster and can then link through to the full documentation if they want to learn more information as opposed to diving into code examples. cc @JackUkleja for a product perspective on this.

Reply:
As a developer, it's common to go to Github first, because that's the code is, which is the ultimate source of truth. Personally I find it quite frustrating when libraries on Github don't show you simple "getting started" examples, and you have to go and look elsewhere, or look in the tests, or whatever.

I do think that the code snippets in Github should not be extensive, and they obviously need to point to the main documentation for more information.

Reply:
>  Alongside the documentation and support help pages

No one goes to the support help pages to get documentation do they?  I am not aware of that. I am only aware of us linking people to support pages from our docs which is where the confusion lies.

>  that make 3 potential sources of truth

Sure, but as Paddy has said, developers come to Ably via Github frequently when they download an SDK.  So the key features and code samples is needed, and we should be encouraging people to the canonical docs at www.ably.io/documentation.

Reply:
>  It might make sense to link through to the actual documentation here if more information is needed.

Regardless I do think it's important that whatever content we provide to developers is understandable on the basis that they may know nothing about Ably.

Reply:
@MarkWoulfeAbly @mattheworiordan I'm not really clear what the next step is here. Do you know what we should link to in the docs, as in a specific page? This discussion doesn't seem to have reached a conclusion.

Feel free to suggest new words and/or push a commit or two to this branch. Thanks!

Reply:
change suggested below